assets/js/app.js

```diff
@@ -57,39 +57,12 @@ $(function() {
 
 $(() => {
   $("#search").submit(function(event){
-    event.preventDefault();
-    $(".search-input-view").show();
-
-    let data =  $('#search-input').val()
-    if (data.length > 0) {
-      const idx = lunr(function () {
-        this.ref('id')
-        this.field('title', {
-          boost: 15
-        })
-        this.field('tags')
-        this.field('content', {
-          boost: 10
-        })
-    
-        for (const key in window.store) {
-          this.add({
-            id: key,
-            title: window.store[key].title,
-            tags: window.store[key].category,
-            content: window.store[key].content
-          })
-        }
-      })
-    
-      // Perform the search
-      const results = idx.search(data)
-      // Update the search title
-      $('#search-title').text('Search Results for ' + '"' + data + '"')
-      // Update the list with results
-      displayResults(results, window.store)
+    if($(".search-input-view").is(":hidden"))
+    {
+      event.preventDefault();
+      $(".search-input-view").show();
```

Comment:
It would make sense for this to be a toggle, so that it can be closed again?

It can be a separate discussion with UX though.

Reply:
We could if the input field is empty. I'd be fine with it either way.  It doesn't appear that the visible input field is hurting the UX currently - I checked on mobile and it doesn't appear to cover anything up when it's visible.